deploy/charts/cert-manager/README.template.md

```diff
@@ -133,6 +133,7 @@ The following table lists the configurable parameters of the cert-manager chart
 | `https_proxy` | Value of the `HTTPS_PROXY` environment variable in the cert-manager pod | |
 | `no_proxy` | Value of the `NO_PROXY` environment variable in the cert-manager pod | |
 | `webhook.replicaCount` | Number of cert-manager webhook replicas | `1` |
+| `webhook.timeoutSeconds` | Seconds the API server should wait the webhook to respond before treating the call as a failure. | `5` |
```

Comment:
Can we keep this on 10 seconds as that is the Kubernetes default?
While it may seem like a lot it might break users, if you need it to be shorter you have an option to change it now